skymap_scanner/server/start_scan.py

```diff
@@ -84,30 +86,14 @@ def __init__(
         self.input_pos_name = input_pos_name
         self.input_time_name = input_time_name
         self.output_particle_name = output_particle_name
-        self.reco_algo = reco_algo.lower()
-
-        # Get Position Variations
-        variation_distance = 20.*I3Units.m
-        if self.reco_algo == 'millipede_original':
-            if cfg.ENV.SKYSCAN_MINI_TEST:
-                self.pos_variations = [
-                    dataclasses.I3Position(0.,0.,0.),
-                    dataclasses.I3Position(-variation_distance,0.,0.)
-                ]
-            else:
-                self.pos_variations = [
-                    dataclasses.I3Position(0.,0.,0.),
-                    dataclasses.I3Position(-variation_distance,0.,0.),
-                    dataclasses.I3Position(variation_distance,0.,0.),
-                    dataclasses.I3Position(0.,-variation_distance,0.),
-                    dataclasses.I3Position(0., variation_distance,0.),
-                    dataclasses.I3Position(0.,0.,-variation_distance),
-                    dataclasses.I3Position(0.,0., variation_distance)
-                ]
-        else:
-            self.pos_variations = [
-                dataclasses.I3Position(0.,0.,0.),
-            ]
+        
+        self.reco_algo: str = reco_algo
+        
+        RecoAlgo: type[RecoInterface] = recos.get_reco_interface_object(reco_algo)
+        
+        self.reco: RecoInterface = RecoAlgo()
```

Comment:
Optional, but I'd combine this into one line since `RecoAlgo` isn't used after this line explicitly

Reply:
It would make sense but I would keep it this way for the sake of readability (especially if we add arguments to the constructor in the future).

Reply:
Sounds good :+1: